client/my-sites/google-my-business/select-business-type/style.scss

```diff
@@ -41,25 +44,33 @@
 	@include breakpoint( "<960px" ) {
 		flex-direction: column-reverse;
 	}
-	
+
 	h1 {
 		color: $gray-dark;
 		font-size: 24px;
 		font-weight: 400;
+		margin: 16px 0;
 	}
 
-	h2 {
+	p {
 		color: $gray-dark;
-		font-size: 16px;
+		font-size: 18px;
 		font-weight: 400;
 	}
 
 	img {
-		max-width: 300px;
+		max-width: 400px;
+		width: 150%;
 		align-self: center;
+
+		@include breakpoint( "<1040px" ) {
```

Comment:
We should try to use > breakpoints when possible to reduce the rendering load for mobiles